src/plugins/gui/providers/kadoProvider.ts

```diff
@@ -613,10 +616,32 @@ export const kadoProvider: FiatProviderFactory = {
             }
             console.log('Launching Kado webview url=' + url.href)
 
+            // If Kado needs to launch the Plaid bank linking widget, it needs it in an external
+            // webview to prevent some glitchiness. When needed, Kado will send an onMessage
+            // trigger with the url to open. The below code is derived from Kado's sample code
+            const onMessage = (data: string) => {
+              try {
+                datelog(`**** Kado onMessage ${data}`)
```

Comment:
is data for sure not sensitive and therefore okay to log?

Reply:
For sure. it's just a url link to redirect the user